backend/siarnaq/api/episodes/challonge.py

```diff
@@ -0,0 +1,95 @@
+# An API-esque module for our usage.
+# Commands here are not very generic (like a good API),
+# and are instead tailored to Battlecode's specific usage,
+# to improve dev efficiency
+
+import json
+import os
+
+import requests
+
+_headers = {
+    "Accept": "application/json",
+    "Authorization-Type": "v1",
+    # TODO refactor this into normal settings var etc
+    "Authorization": os.getenv("CHALLONGE_API_KEY"),
+    "Content-Type": "application/vnd.api+json",
+    # requests' default user agent causes Challonge's API to crash.
+    "User-Agent": "",
+}
+
+AUTH_TYPE = "v1"
+URL_BASE = "https://api.challonge.com/v2/"
+
+
+def set_api_key(api_key):
+    """Set the challonge.com api credentials to use."""
+    _headers["Authorization"] = api_key
+
+
+def create_tour(tour_url, tour_name, is_private=True, is_single_elim=True):
```

Comment:
Naming in general: please don't abbreviate. Use `tournament` instead of `tour` throughout.